LEAF_Request_Portal/js/formSearchMultisite.js

```diff
@@ -0,0 +1,1111 @@
+
+          posSel.setSelectHandler(function () {
+            $("#" + prefixID + "widgetMat_" + widgetID).val(posSel.selection);
+            $("#" + posSel.prefixID + "input").val("#" + posSel.selection);
+          });
+          posSel.setResultHandler(function () {
+            $("#" + prefixID + "widgetMat_" + widgetID).val(posSel.selection);
+          });
+          posSel.initialize();
+        },
+      });
+    } else {
+      let posSel = new positionSelector(prefixID + "widgetPos_" + widgetID);
+      posSel.apiPath = orgchartPath + "/api/";
+      posSel.rootPath = orgchartPath + "/";
+
+      posSel.setSelectHandler(function () {
+        $("#" + prefixID + "widgetMat_" + widgetID).val(posSel.selection);
+        $("#" + posSel.prefixID + "input").val("#" + posSel.selection);
+      });
+      posSel.setResultHandler(function () {
+        $("#" + prefixID + "widgetMat_" + widgetID).val(posSel.selection);
+      });
+      posSel.initialize();
+    }
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function createGroupSelectorWidget(widgetID) {
+    if (typeof groupSelector == "undefined") {
+      $("head").append(
+        '<link type="text/css" rel="stylesheet" href="' +
+          orgchartPath +
+          '/css/groupSelector.css" />'
+      );
+      $.ajax({
+        type: "GET",
+        url: orgchartPath + "/js/groupSelector.js",
+        dataType: "script",
+        success: function () {
+          let grpSel = new groupSelector(prefixID + "widgetGrp_" + widgetID);
+          grpSel.apiPath = orgchartPath + "/api/";
+          grpSel.rootPath = orgchartPath + "/";
+
+          grpSel.setSelectHandler(function () {
+            $("#" + prefixID + "widgetMat_" + widgetID).val(grpSel.selection);
+            $("#" + grpSel.prefixID + "input").val("group#" + grpSel.selection);
+          });
+          grpSel.setResultHandler(function () {
+            $("#" + prefixID + "widgetMat_" + widgetID).val(grpSel.selection);
+          });
+          grpSel.initialize();
+        },
+      });
+    } else {
+      let grpSel = new groupSelector(prefixID + "widgetGrp_" + widgetID);
+      grpSel.apiPath = orgchartPath + "/api/";
+      grpSel.rootPath = orgchartPath + "/";
+
+      grpSel.setSelectHandler(function () {
+        $("#" + prefixID + "widgetMat_" + widgetID).val(grpSel.selection);
+        $("#" + grpSel.prefixID + "input").val("group#" + grpSel.selection);
+      });
+      grpSel.setResultHandler(function () {
+        $("#" + prefixID + "widgetMat_" + widgetID).val(grpSel.selection);
+      });
+      grpSel.initialize();
+    }
+  }
+
+  /**
+   * Render the query match condition's input type for dropdown and radio fields
+   * @param widgetID
+   * @param options <select> html section matching the widgetID
+   * @memberOf LeafFormSearchMultisite
+   */
+  function renderSingleSelectInputType(widgetID, options) {
+    switch ($("#" + prefixID + "widgetCod_" + widgetID).val()) {
```

Comment:
we've been using toLower on any text fed to a switch and then putting all the putting all the case in lower.

At least for ones that we're creating or small ones that are an easy fix.  I see another below that you brought over from other code.  We've been kinda letting those slide for now.

Reply:
In this situation I'd want to keep it as-is because the variable being fed into this uses an SQL all-caps convention, which should be consistent everywhere. The input is controlled by us, and if there's a case sensitivity issue, it would mean the upstream source should be fixed to ensure consistency.